<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(248, 236, 209);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame#frame_7{
background-color:rgb(127, 124, 130);
border-top-left-radius:20px;
border-top-right-radius:20px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>Tw Cen MT</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel#label_4{
background-color: none;
color: rgb(243, 241, 245);
}</string>
                 </property>
                 <property name="text">
                  <string>Login</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true">QFrame#frame_2{
background-color:rgb(243, 241, 245);
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QFrame" name="frame_11">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame#frame_11{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="font">
                     <font>
                      <family>Tw Cen MT</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel#label_5{
background-color:none;
color:rgb(80, 80, 80);
}</string>
                    </property>
                    <property name="text">
                     <string>Lakukan scan KTM dan tekan tombol refresh untuk memunculkan ID KTM anda pada layar</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_12">
                 <property name="styleSheet">
                  <string notr="true">QFrame#frame_12{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame_14">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QFrame#frame_14{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <item>
                      <widget class="QLabel" name="label_8">
                       <property name="font">
                        <font>
                         <family>Tw Cen MT</family>
                         <pointsize>14</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QLabel#label_8{
background-color:none;
color:rgb(80, 80, 80);
}</string>
                       </property>
                       <property name="text">
                        <string>ID :</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_15">
                    <property name="styleSheet">
                     <string notr="true">QFrame#frame_15{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <item>
                      <widget class="QLabel" name="label_11">
                       <property name="font">
                        <font>
                         <family>Tw Cen MT</family>
                         <pointsize>12</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QLabel#label_11{
background-color:none;
color:rgb(80, 80, 80);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="wordWrap">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_9">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame#frame_9{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <family>Tw Cen MT</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel#label{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_13">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame#frame_13{
background-color:none;
color:rgb(255, 84, 124);
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>120</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>120</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Tw Cen MT</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton#pushButton_3{
background-color:rgb(127, 124, 130);
border-top-left-radius:20px;
border-bottom-left-radius:20px;
border-top-right-radius:20px;
border-bottom-right-radius:20px;
color:rgb(255, 255, 255);

}

QPushButton#pushButton_3:hover{
background-color:rgb(159, 159, 159);
}

QPushButton#pushButton_3:pressed{
padding-left:5px;
padding-top:5px;
}</string>
                    </property>
                    <property name="text">
                     <string>Refresh</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame#frame_8{
background-color:rgb(127, 124, 130);
border-bottom-left-radius:20px;
border-bottom-right-radius:20px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_10">
              <property name="styleSheet">
               <string notr="true">QFrame#frame_10{
background-color:rgb(127, 124, 130);
border-radius:20px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>Tw Cen MT</family>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel#label_2{
background-color: none;
color: rgb(243, 241, 245);
}</string>
                 </property>
                 <property name="text">
                  <string>Login to edit</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>550</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton{
background-color:qlineargradient(spread:pad, x1:0.12, y1:0.449, x2:1, y2:1, stop:0 rgba(133, 88, 111, 255), stop:1 rgba(172, 125, 136, 255));
border-top-left-radius:20px;
border-bottom-left-radius:20px;
border-top-right-radius:20px;
border-bottom-right-radius:20px;
color:rgb(255, 255, 255);

}

QPushButton#pushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.0802273, y1:0.375, x2:1, y2:1, stop:0 rgba(172, 125, 136, 255), stop:1 rgba(222, 182, 171, 255));
}

QPushButton#pushButton:pressed{
padding-left:5px;
padding-top:5px;
}</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_2{
background-color:qlineargradient(spread:pad, x1:0.12, y1:0.449, x2:1, y2:1, stop:0 rgba(133, 88, 111, 255), stop:1 rgba(172, 125, 136, 255));
border-top-left-radius:20px;
border-bottom-left-radius:20px;
border-top-right-radius:20px;
border-bottom-right-radius:20px;
color:rgb(255, 255, 255);

}

QPushButton#pushButton_2:hover{
background-color:qlineargradient(spread:pad, x1:0.0802273, y1:0.375, x2:1, y2:1, stop:0 rgba(172, 125, 136, 255), stop:1 rgba(222, 182, 171, 255));
}

QPushButton#pushButton_2:pressed{
padding-left:5px;
padding-top:5px;
}</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
